dotenv-linter/tests/common/test_dir.rs

```diff
@@ -14,20 +15,32 @@ use std::str::from_utf8;
 /// Use to test commands in temporary directories
 pub struct TestDir {
     current_dir: TempDir,
+    envs: Option<HashMap<String, String>>,
 }
 
 impl TestDir {
     /// Generate a new TestDir
     pub fn new() -> Self {
         Self {
             current_dir: tempdir().expect("create testdir"),
+            envs: None,
```

Comment:
```suggestion
            envs: Default::default(),
```
It's alternative to use `HashMap::new()`.